src/rules/maxLineLengthRule.ts

```diff
@@ -71,11 +76,73 @@ export class Rule extends Lint.Rules.AbstractRule {
                 // (and thus we are not actually over the limit).
                 const ruleFailure = new Lint.RuleFailure(sourceFile, from, to - 1, errorString, this.getOptions().ruleName);
                 if (!Lint.doesIntersect(ruleFailure, disabledIntervals)) {
-                    ruleFailures.push(ruleFailure);
+                    let report = true;
+                    if (ruleArgs.indexOf("ignoreUrls") > -1) {
```

Comment:
I think this can only be used on a walker. I was going to move the logic in the `apply` function to the walker so I'll be able to do this there.